submissions/NwagboJohn/HtmlTask/signUp.html

```diff
@@ -0,0 +1,34 @@
+
+   <!DOCTYPE html>
+   <html>
+	<head>
+		<title>Johnie store</title>
+	<meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./css/style.css">

```

Comment:
your `href` here is pointing to a `css` folder that does not exist. you should change it to 
```<link rel="stylesheet" href="style.css">```
Since the html and css file are in the same folder, it should be fine.